internal/app/tfsec/checks/azu008.go

```diff
@@ -0,0 +1,59 @@
+package checks
+
+import (
+	"fmt"
+
+	"github.com/tfsec/tfsec/internal/app/tfsec/parser"
+	"github.com/tfsec/tfsec/internal/app/tfsec/scanner"
+)
+
+const AZUAKSAPIServerAuthorizedIPRanges scanner.RuleCode = "AZU008"
+const AZUAKSAPIServerAuthorizedIPRangesDescription scanner.RuleSummary = "Ensure AKS has an API Server Authorized IP Ranges enabled"
+const AZUAKSAPIServerAuthorizedIPRangesExplanation = `
+The API server is the central way to interact with and manage a cluster. To improve cluster security and minimize attacks, the API server should only be accessible from a limited set of IP address ranges.
+`
+const AZUAKSAPIServerAuthorizedIPRangesBadExample = `
+resource "azurerm_kubernetes_cluster" "my-aks-cluster" {
+
+}
+`
+const AZUAKSAPIServerAuthorizedIPRangesGoodExample = `
+resource "azurerm_kubernetes_cluster" "my-aks-cluster" {
+    api_server_authorized_ip_ranges = [
+		x.x.x.x/32
```

Comment:
Would be good to use a "real" value here as the tests parse this example as actual terraform
```suggestion
		"1.2.3.4/32"
```